blueprints/ember-redux/files/__root__/reducers/index.js

```diff
@@ -0,0 +1,18 @@
+import { combineReducers } from 'redux';
+
+/**
+ * This is the master reducer, it partitions the jobs amounts
+ * one or more other reducers which take on responsibility for
+ * a discrete part of the state tree.
+ *
+ * Use the "ember generate reducer" command to create additional
+ * reducers.
+ */
+
+// import foo from './foo';
+// import bar from './bar';
+
+export default combineReducers({
```

Comment:
> That said, I've noticed the trend/pattern and think others would find this valuable. Overall it's likely I'll work up a simplified version of this pull request and get a simple/correct blueprint into the project.

I'm fairly sure the style that I'm using is consistent with the latest recommendations Dan Abramov. What about my style is "incorrect"?